src/modules/function.js

```diff
@@ -89,6 +89,14 @@ function tasksDisplay() {
 }
 tasksDisplay();
 
+const clearCompleted = document.querySelector('.clear-completed');
+clearCompleted.addEventListener('click', () => {
+  const storage = JSON.parse(localStorage.getItem('tasks'));
+  const notCompleted = storage.filter((item) => item.completed === false);
+  localStorage.setItem('tasks', JSON.stringify(notCompleted));
+  window.location.reload();
+});
+
```

Comment:
- [ ] Kindly reset the `index` key of each array when you clear all completed tasks. You can call the `resetIndex()` function here. 👍